enhancements/builds/disable-jenkins-pipeline.md

```diff
@@ -0,0 +1,303 @@
+---
+title: disable-jenkins-pipeline
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@gabemontero"
+  - "@akram"
+approvers:
+  - "@bparees"
+  - "@sbose78"
+  - "@derekwaynecarr"
+creation-date: 2021-04-21
+last-updated: 2021-04-23
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Disable Jenkins Pipeline Strategy By Default
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenShift 3.0 included a way to invoke a Jenkins pipeline from an OpenShift build.
+The status of the build on the OpenShift cluster would reflect the state of the pipeline in Jenkins.
+This was the first feature to support continuous integration/delivery processes on OpenShift.
+
+As OpenShift and Kubernetes have evolved, so has cloud-native CI/CD.
+Tekton and its downstream distribution - OpenShift Pipelines - provide a way of running CI/CD processes natively on Kubernetes.
+The Jenkins pipeline strategy was officially deprecated in OpenShift 4.1, but at that time there was no meaningful replacement.
+We now have that meaningful replacement with the general availability of OpenShift Pipelines.
+
+To discourage use of the Jenkins pipeline strategy, this proposal will remove Jenkins from the OCP payload and disable runs of Jenkins Pipeline builds by default.
```

Comment:
i'm not sure these two things need to be coupled together.  Is there any value in doing one first and then the other later?
(e.g. putting pipelines behind a featuregate is easy and low risk/impact to customers.  Removing the image from the payload is somewhat higher risk, particularly for disconnected clusters)

in any case removing the payload content doesn't really discourage use of the strategy.  I'd say it's more the case that "as we discourage its use, we feel more empowered to remove the content from the payload"  (hence my suggestion that we gate pipelines first, and then remove the payload content later)

it doesn't have to be done separately, just throwing it out there as a risk mitigation option.

Reply:
Split removal from the payload into its own EP - #841